server/app/views/v1/host_nodes/_host_node.json.jbuilder

```diff
@@ -1,4 +1,5 @@
 json.id node.node_id
+json.token node.token
```

Comment:
This does mean that `POST /v1/grids/:grid/nodes` no longer returns the `{"token": ...}`, so any provisioning tool will have to do an extra `GET /v1/nodes/:grid/:name/token` request for it.